2.train-VAE/optimize_utils.py

```diff
@@ -1,96 +1,49 @@
 # THIS CODE WAS SOURCED FROM THE FOLLOWING URL: https://github.com/broadinstitute/cell-painting-vae/blob/master/scripts/optimize_utils.py
 
-from pyexpat import model
-from keras import backend as K
-from keras_tuner import HyperModel
-from keras_tuner.tuners import BayesianOptimization
-import sys
-from vae import VAE
-import random
+import torch
```

Comment:
I think you can delete line 1 now (it departs from this URL now)

Also, as i'm looking through this function, it doesn't look like it belongs in the `optimize_utils.py` file (or at least it doesn't look like you'll be using it exclusively for optimization. It looks like it will work well for training and any other time you'd train/apply a VAE)